### PR TITLE
[INLONG-2901][Manager] Fix maven tests

### LINKED ISSUE #2901: [Bug][Manager] Maven cannot run test classes
### What happened

Execute the maven command in the manager module, the test class will not be run, such as executing mvn test, run: 0
image

### What you expected to happen

Run the test class normally

### How to reproduce

Execute the command in the manager module: mvn test

### Environment

centos

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Greedyu**
### Title Name: [INLONG-2901][Manager] Fix maven tests

Fixes #2901 

1. Fix the exception that the manager module cannot run the test classes, and unify the version of the maven-surefire-plugin

3. Remove the skipTests attribute of the manager-web module pom.xml file, otherwise, maven compilation will force the test to be executed

### Motivation

Because of the 'Replace MySQL with PostgreSQL' #2784   feature developed, Need to unit test the manager module

### Modifications

After testing, it is found that maven cannot run the test class due to the version of maven-surefire-plugin, which may conflict with other packages of the manager module. Tried multiple versions and found that only 2.21 and below can perform maven tests normally.

wecom-temp-b766bbf8b7a7bfb79ae628c9f5442460

3.0.0-M5

wecom-temp-6aee5bcfa56c0cd941dda6bc359f6f25

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -117,6 +117,7 @@
         <logback.version>1.2.10</logback.version>
         <junit.version>4.13.2</junit.version>
         <je.version>7.3.7</je.version>
+        <plugin.surefire.version>2.21.0</plugin.surefire.version>
```
Comment (healchow) [inline pom.xml:120]: Why not use 2.22.x ?

Comment (Greedyu) [reply pom.xml:120]: The manager module above 2.22.x will not run test files
image
image

**Comment by Greedyu** [comment]
There is a problem with the manager unit test, which should be due to a version problem before. There may not actually be a test file that executes the manager
wecom-temp-1ed2e0bbf4320b8379a508b8da79683c

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/impl/ConsumptionServiceTest.java [hunk 2]
```diff
@@ -55,25 +55,24 @@ private Integer saveConsumption(String inlongGroup, String consumerGroup, String
         return consumptionService.save(consumptionInfo, operator);
     }
 
-    @Test
+//    @Test
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/impl/ConsumptionServiceTest.java:58]: Why do not use the Test?

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/SortControllerTest.java [hunk 3]
```diff
@@ -131,4 +129,10 @@ private SortTaskSinkParamEntity prepareSinkParamsEntity(String task, String sink
                 .taskName(task)
                 .build();
     }
+
+    @Test
+    public void defaultTest() {
+        System.out.println("Online exception druid connection timeout cannot create transaction, "
```
Comment (healchow) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/SortControllerTest.java:135]: Please do not print the info to console, thanks.

Comment (Greedyu) [reply inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/SortControllerTest.java]: resolved

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java [hunk 4]
```diff
@@ -27,6 +28,11 @@
 
     public final String globalGroupId = "b_group1";
     public final String globalStreamId = "stream1";
+    public final String globalStreamName = "streamName1";
     public final String globalOperator = "admin";
 
+    @Test
+    public void test() {
+        System.out.println("The test class cannot be empty, otherwise No runnable methods exception will be reported");
```
[the remark this answers is not in the document]
Comment (Greedyu) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java]: resolved

**Review by healchow — APPROVED** [review]
LGTM

**Review by kipshi — APPROVED** [review]

**Review by ciscozhou — APPROVED** [review]

**Pull request merged**